e2e/e2e_test.go

```diff
@@ -246,14 +246,14 @@ func macvlanNetworkWithoutIPAM(networkName string, namespaceName string) *nettyp
 
 func macvlanNetworkWitStaticIPAM(networkName string, namespaceName string) *nettypes.NetworkAttachmentDefinition {
 	macvlanConfig := fmt.Sprintf(`{
-        "cniVersion": "1.0.0",
+        "cniVersion": "0.3.0",
         "disableCheck": true,
         "name": "%s",
         "plugins": [
 			{
 				"type": "macvlan",
 				"capabilities": { "ips": true },
-				"master": "eth1",
+				"master": "eth0",
```

Comment:
Ideally it should be possible to execute e2e test suite on an arbitrary cluster. e.g. if I want to use this as a conformance suite on a baremetal cluster, the NIC name would be probably different.

Would you consider making this name configurable through an environment variable?

Reply:
Done. Good idea.
@phoracek please review.